tests/manifests/test-cluster-on-pvc-encrypted.yaml

```diff
@@ -14,7 +14,7 @@ spec:
           requests:
             storage: 5Gi
   cephVersion:
-    image: quay.io/ceph/ceph:v16
+    image: quay.io/ceph/ceph:v17.2.3
```

Comment:
We can't use v17 in these older branches since they don't support Quincy. We need a Pacific version v16.2.x, looking for the exact version that should work...

Reply:
Based on [this comment](https://github.com/rook/rook/pull/11646#issuecomment-1425290903), v16.2.10 should work.

Reply:
seems like all encryption pvcs tests are failing and cluster based on older Kubernetes versions

Reply:
this is for 4.11 which means rook 1.9 so lot has changes in the ci and code, we can't BP all the prs right?
Also, we need similar to 4.10 too.